src/plugins/vis_default_editor/public/_default.scss

```diff
@@ -9,6 +9,7 @@
 
 .visEditor__collapsibleSidebar {
   flex-grow: 1;
+  overflow-y: hidden;
```

Comment:
Hey, @stratoula! While investigating this issue, it appeared to me that the scrollbar appearing when a bucket is added was directly related to the `p#jsonEditorDescription` element that appears within the "Advanced" accordion in the "Buckets" panel.

![image](https://user-images.githubusercontent.com/3884767/170533292-1fea4a35-7a37-4b2a-8632-a1ac335f048c.png)

I noticed that removing this element from the markup also corrected the erroneous scrollbar. It seems that something about the placement of this paragraph within the markup while using the `euiScreenReaderOnly` mixin is causing some unexpected behavior. Making the element's parent container (`.euiFormRow__fieldWrapper`) `position: relative;` does appear to correct the issue.

However, in looking at the text for this description, I noticed that it is the exact same text that appears in the icon tooltip next to the "JSON input" label already. Aren't those tooltips already screen reader accessible? And if so, why the redundant message for screen readers? If no reason, we can likely remove this element altogether.

Reply:
@MichaelMarcialis you are right! I removed the `euiScreenReaderOnly ` as the same information is given by the tooltip. I also tested it with a screen reader Chrome plugin and it works fine. Thanx for the tip!